src/sonic-config-engine/minigraph.py

```diff
@@ -229,9 +230,12 @@ def parse_dpg(dpg, hname):
                     print >> sys.stderr, "Warning: ACL " + aclname + " is attached to a Vlan interface, which is currently not supported"
                 elif port_alias_map.has_key(member):
                     acl_intfs.append(port_alias_map[member])
+                elif member.lower() == 'erspan':
```

Comment:
add comment here? If it is erspan, then attach the acl to all front panel ports.